dev/src/index.ts

```diff
@@ -703,14 +703,12 @@ export class Firestore {
    *   console.log(`Second document: ${JSON.stringify(docs[1])}`);
    * });
    */
-  getAll(
-      documentRef: DocumentReference,
-      ...moreDocumentRefsOrReadOptions: Array<DocumentReference|ReadOptions>):
+  getAll(...documentRefsOrReadOptions: Array<DocumentReference|ReadOptions>):
```

Comment:
Thanks for doing this :) One nit from my end: Can you also update the JSDoc for these methods?

Reply:
Updated and verified. I will also include an additional comment regarding the required order of arguments.